Gemfile.jruby-1.9.lock.release

```diff
@@ -571,7 +570,7 @@ GEM
     sawyer (0.6.0)
       addressable (~> 2.3.5)
       faraday (~> 0.8, < 0.10)
-    sequel (5.1.0)
+    sequel (5.2.0)
```

Comment:
Brought in by the jdbc input plugin fix for https://github.com/elastic/support-dev-help/issues/2695